src/components/TimePicker/TimePicker.tsx

```diff
@@ -19,24 +19,17 @@ import * as DeviceCapabilities from '@libs/DeviceCapabilities';
 import CONST from '@src/CONST';
 import setCursorPosition from './setCursorPosition';
 
-const propTypes = {
-    /** Refs forwarded to the TextInputWithCurrencySymbol */
-    forwardedRef: refPropTypes,
+type MinHrRefs = {hourRef: TextInput | null; minuteRef: TextInput | null};
```

Comment:
I think ideally we should use a non-abbreviated name here like `MinuteHourRefs`  or `MinuteAndHourRefs`